app/src/main/java/com/github/scribeWizTeam/scribewiz/models/ChallengeModel.kt

```diff
@@ -88,12 +89,12 @@ data class ChallengeModel(
         }
     }
 
-    fun addSubmission(recordId: String, userId: String) {
+    fun addSubmission(recordId: String, userId: String) : Task<Void> {
```

Comment:
Once merged, this should call uploadFileToDatabase from my PR to actually store the file in the server

Reply:
Are you sure it has to be done here ? It's purpose is to link the Challenge with an ID (this ID has to be the one of an already uploaded music note)